README.md

```diff
@@ -46,7 +46,7 @@ Let's walk through an example, non-existent, exercise, which we'll call
 
 ### Exercise configuration
 
-An exercise is [configured](https://github.com/exercism/docs/blob/master/language-tracks/configuration/exercises.md)
+An exercise is [configured](https://exercism.org/docs/building/tracks/config-json)
```

Comment:
I would recommend to remove the link here entirely. It doesn't really add value when the link is provided below with clear text that this is where to read more.

Reply:
Done, such link removed